app/screens/phone-auth-screen/phone-input.tsx

```diff
@@ -332,3 +338,9 @@ export const PhoneInputScreen: React.FC = () => {
     </Screen>
   )
 }
+
+const phoneNumberIsLocalTestAccount = (phone: string) => {
+  // TODO: extract the list from https://github.com/GaloyMoney/galoy/blob/main/galoy.yaml#L38
+  if (phone === "+16505554321") return true
```

Comment:
This is a real phone number and it will prevent them from signing in on prod (although very small chance they have an account). Its probably cleaner to bypass based on the GaloyInstace from config rather than the phone number.

Reply:
I think 555 are dedicated number for special use case (including for TV shows) so I think the assumption to get a user from that account is... null? 

https://en.wikipedia.org/wiki/555_(telephone_number)

Reply:
I'm not sure to understand how you would do it otherwise? the thing is that if it's not a real number, it's been considered invalid by the component and we can't move to the next screen

Reply:
See above.